src/mapping/config/MappingConfiguration.cpp

```diff
@@ -81,8 +81,11 @@ MappingConfiguration::MappingConfiguration(
   }
   {
     XMLTag tag(*this, VALUE_RBF_GAUSSIAN, occ, TAG);
-    tag.setDocumentation("Local radial-basis-function mapping based on the Gaussian RBF with a cut-off threshold.");
-    tag.addAttribute(attrShapeParam);
+    tag.setDocumentation("Local radial-basis-function mapping based on the Gaussian RBF using a cut-off threshold.");
+    tag.addAttribute(makeXMLAttribute<double>(ATTR_SHAPE_PARAM, 0.)
+                         .setDocumentation("Specific shape parameter for RBF basis function."));
+    tag.addAttribute(makeXMLAttribute(ATTR_SUPPORT_RADIUS, std::numeric_limits<double>::infinity())
+                         .setDocumentation("Support radius of each RBF basis function (global choice)."));
```

Comment:
Why creating the attributes here? The other cases are handled differently.

Reply:
We need to set default values, otheriwse, we break compatibility.

Reply:
That's connected to what I wrote above:

> I could imagine that the way you use default values in the config is a bit error prone as they are no real default values, but "uninitialized" values.

We could instead check whether the user defined the attributes or not, right? And then throw errors if none or both are defined.